### ISSUE TITLE
Confusion in training and validation data sets

### ISSUE THREAD

**Issue by jiaweichaojwc** [tianrun-chen/SAM-Adapter-PyTorch#56]
When I was training my dataset, the problem of training the validation set as the training set came up.
![image](https://github.com/tianrun-chen/SAM-Adapter-PyTorch/assets/113352745/cc2d5568-3b97-4491-b928-8db3066b8fea)
And when I train with a card, this problem disappears again, can you explain to me why? Thank you in advance for your excellent work, and hope to have time for a reply
